cmd/flowrunner/testdata/flows/subflow_loop_test.json

```diff
@@ -19,10 +19,10 @@
         {
           "action_uuid": "300f02ba-e0b5-4991-bed6-4c240cdb8743",
           "event": {
-            "contact_uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3",
             "created_on": "2000-01-01T00:00:00.000000000-00:00",
             "flow_uuid": "a8d27b94-d3d0-4a96-8074-0f162f342195",
-            "type": "flow_entered"
+            "parent_run_uuid": "",
```

Comment:
This test plays out a bit differently now since we don't use enter/exit events to detect loops but lean on the same node-level tracking in the main loop
- Parent flow runs, triggers Child Flow
- Child Flow runs, triggers Parent Flow again (run no. 2)
- Parent Flow (run no. 2) tries to start but immediately hits fatal error because of loop detection
- Child Flow is errored and exited
- Parent flow (run no. 1) is errored and exited
- Session is errored and exited